sechub-doc/src/docs/asciidoc/documents/shared/configuration/executor_config.adoc

```diff
@@ -0,0 +1,229 @@
+// SPDX-License-Identifier: MIT
+[[section-executor-configuration]]
+=== Executor configuration
+{sechub} uses `product executors` to call security products by dedicated `product adapters`.
+Each used security product has at least one executor configuration.
+
+Inside such a configuration, we can define properties and parameters which are used when it comes
+to {sechub} job execution. The data is evaluated and used by product executors at runtime and no server restart is necessary.
+
+So for example you can easily change a base URL of a product location and it will automatically be
+used by next job execution.
+
+[WARNING]
+====
+`NESSUS` and `NETSPARKER` product executors does currently not use
+all properties and parameters but needs still some dedicated ENV entries! But you have
+to define a configuration - and a profile - for those products as well!
+
+====
+
+==== Data structure
+
+===== Name
+Just the name of the executor configuration. Can be changed
+
+===== Enabled
+Only when the executor configuration is enabled, it will be used (of courseit has also
```

Comment:
a space is missing: `of courseit` it should be `of course it`.

Reply:
Done